store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/operation/hdfs/handler/job/tool/ImportElementsToAccumuloTool.java

```diff
@@ -59,7 +69,10 @@ public int run(final String[] strings) throws Exception {
         fs.delete(new Path(inputPath + "/_SUCCESS"), false);
 
         // Set all permissions
-        IngestUtils.setDirectoryPermsForAccumulo(fs, new Path(inputPath));
+        if (!options.containsKey(AccumuloProperties.HDFS_SKIP_PERMISSIONS) ||
```

Comment:
You could get a null pointer exception here if the value of the option is null. It is probably better to do:

```java
if(!Boolean.parseBoolean(options.get(AccumuloProperties.HDFS_SKIP_PERMISSIONS))) {
    IngestUtils.setDirectoryPermsForAccumulo(fs, new Path(inputPath));
}


```